Rover/UIImageView+RVAsset.swift

```diff
@@ -43,7 +43,19 @@ extension UIImageView {
             
             guard let data = data else { return }
             
-            self.image = UIImage(data: data)
+            if let img = UIImage(data: data) {
+                self.image = img
+            } else {
```

Comment:
This assumes that if it could not create an image from the data that the response payload must be a data URI. This is very specific to RMN's use case. 

The feature we added in #173 was to support data URIs in the JSON payload . We check the URL scheme and if it is indeed a valid data URI then we bypass the Asset manager (don't need to download anything) and render the image directly. That use case is much more general purpose and potentially useful to other users. I'm not sure we can accept this PR as it is.